src/Avalonia.Visuals/Media/Fonts/FamilyNameCollection.cs

```diff
@@ -24,7 +24,10 @@ public FamilyNameCollection(IEnumerable<string> familyNames)
 
             var names = new List<string>(familyNames);
 
-            if (names.Count == 0) throw new ArgumentException($"{nameof(familyNames)} must not be empty.");
+            if (names.Count == 0)
```

Comment:
Automatically formatting the file might not be a good idea :)
Unrelated changes such as that one should be avoided IMO.

Reply:
That is not automated that was my intention to fix the code style. I don't see why we shouldn't fix code style when we work on something. The code base is so big I don't think anyone ever puts its hands on this otherwise.